spec/services/hyrax/workflow/workflow_importer_spec.rb

```diff
@@ -42,8 +46,67 @@
         result = described_class.generate_from_json_file(path: path)
       end.to change { Sipity::Workflow.count }.by(1)
       expect(result).to match_array(kind_of(Sipity::Workflow))
+      expect(described_class.load_errors).to be_empty
       expect(result.first.label).to eq "This is the label"
       expect(result.first.description).to eq "This description could get really long"
     end
   end
+  context "when I load twice" do
+    let!(:workflow1) { described_class.generate_from_json_file(path: path)[0] }
+    let(:workflow2) { described_class.generate_from_json_file(path: path)[0] }
+    let(:workflow2_errors) { described_class.load_errors }
+    it "creates the same results" do
+      expect(workflow2).to eq(workflow1)
+      expect(workflow2_errors).to be_empty
+    end
+    context "When the json changes" do
+      let(:workflow_name) { "awsome workflow" }
+      let(:action_name)  { "awesome action" }
+      let(:state_name)   { "awesome state" }
+      let(:second_path) { double(read: json2) }
+      let(:workflow2) { described_class.generate_from_json_file(path: second_path)[0] }
```

Comment:
Would you consider using `#first` here instead of `[0]`? I find `first` harder to miss when I scan this with my eyes.